presto-main/src/main/java/com/facebook/presto/sql/gen/PageFunctionCompiler.java

```diff
@@ -13,77 +13,32 @@
  */
 package com.facebook.presto.sql.gen;
 
-import com.facebook.presto.bytecode.BytecodeBlock;
-import com.facebook.presto.bytecode.BytecodeNode;
-import com.facebook.presto.bytecode.ClassDefinition;
-import com.facebook.presto.bytecode.FieldDefinition;
-import com.facebook.presto.bytecode.MethodDefinition;
-import com.facebook.presto.bytecode.Parameter;
-import com.facebook.presto.bytecode.ParameterizedType;
-import com.facebook.presto.bytecode.Scope;
-import com.facebook.presto.bytecode.Variable;
+import com.facebook.presto.bytecode.*;
```

Comment:
Can you rebase and squash the unrelated commits?

Reply:
Also, please follow: https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines
If you are using IntelliJ (which I highly recommend), definitely import the style file from here: https://github.com/airlift/codestyle.

Reply:
I thought I had the airift code style. I will doublecheck